components/ContactsItem.tsx

```diff
@@ -7,9 +7,9 @@ interface Props {
 
 export const ContactsItem: FC<Props> = ({ title, text, children }) => {
   return (
-    <div className="contacts__item">
-      <div className="item__title">{title}</div>
-      <div className="item__text">{text}</div>
+    <div className="contacts-item">
+      <h1 className="item__title">{title}</h1>
```

Comment:
The `h1` tag is for the most important title, only can exists `h1` per page

for this case, it is better to use a `h3` or `h4`